sh/tmuxcomplete

```diff
@@ -27,7 +27,8 @@
 # Words including 2000 lines of history per pane
 #     sh tmuxcomplete -c '-S -2000'
 
-if ! tmux info &> /dev/null; then
```

Comment:
To be honest I'm not quite sure. I'm not able to reproduce the issue, but this change seemed to work for @wikimatze. Feel free to join the discussion in #63. I'd like to get to the bottom of this as well.